Mathlib/GroupTheory/Coset.lean

```diff
@@ -371,9 +371,7 @@ theorem induction_on {C : α ⧸ s → Prop} (x : α ⧸ s) (H : ∀ z, C (Quoti
 instance : Coe α (α ⧸ s) :=
   ⟨mk⟩
 
-@[to_additive (attr := elab_as_elim)]
-theorem induction_on' {C : α ⧸ s → Prop} (x : α ⧸ s) (H : ∀ z : α, C z) : C x :=
-  Quotient.inductionOn' x H
+@[to_additive (attr := deprecated (since := "2024-08-04"))] alias induction_on' := induction_on
```

Comment:
I know this has been merged now but did we just lose an `elab_as_elim` attribute on the primed declaration?

Reply:
Good catch, though it probably doesn't matter much since this is now deprecated.